_posts/2019-08-28-mentor-spotlight-katharine-hyatt.markdown

```diff
@@ -0,0 +1,60 @@
+For Julia, I usually work on writing and maintaining libraries to add high-level features. Some things that I have worked in in the past are wrappers for video libraries, and libraries for linear algebra and random number generation, all of which interface with main Julia API. 
+
+### How did you get started in tech? 
+I had no programming experience before I started my undergraduate degree in Physics. In my first year, I took a required class called Introduction to Computational Physics, where I first learned C++. I was pretty good at it and went on to work with my professor from this class as an Undergraduate Research Intern later in my college career. You could say I went from freshman with no idea about Computer Science to mentoring students at HackIllinois. 
+
+### What are some of the projects you are working on currently?
+I primarily work with the Julia ecosystem, with GPU packages more specifically. I think the Julia ecosystem is a viable choice for this as it offers native support for CUDA kernels. I'm currently working on a new sort of [module that deals with High Dimensional Tensors](https://github.com/ITensor/ITensors.jl). We conducted supporting research in the context of Multi-Dimensional Linear Algebra, and I'm happy to say that the project is almost completed and will be useful in fields such as Machine Learning. 
+
+### What is open source/FOSS to you? 
+Open Source holds a lot of significance for me, both personally and professionally. In my industry, there has been a push for open source elements recently to minimize duplication of effort. There might be situations where different groups are solving the same problem but still not getting the same answer legibly. Utilizing open source resources in such situations avoids teams working to reinvent the wheel and also makes it easy to identify problems during code audits.
+
+For me, open source is also a way to give back to the community. What is truly inspiring is that a small contribution by a single person has the potential to pay big dividends for the community at large. 
+    
+### Why do you contribute to open source? 
+When working on algorithm development, my team and I deal with a lot of moving parts and can go months without knowing whether the project is going to work or not. In such times, merging a pull request is something to fall back on as it is somewhat soothing for me to go back home and add two lines to GitHub and seeing everything work.
+
+Working on open source projects has also professionally beneficial for me. My peers and other industry professionals have come to recognize me as the "Julia person". My work has also led me to receive invitations to speak at conferences that are not just focussed on Physics and helped me explore fields broader than my immediate research interests. Having the experience of working on open source projects also makes it easier for me to find professional endeavors in the industry if I decide to explore areas outside of academia.
```

Comment:
"Working on open source projects has also **been** professionally beneficial for me."